volume/mounts/mounts.go

```diff
@@ -108,20 +108,16 @@ func (m *MountPoint) Setup(mountLabel string, rootIDs idtools.Identity, checkFun
 			return
 		}
 
-		var sourcePath string
-		sourcePath, err = filepath.EvalSymlinks(m.Source)
-		if err != nil {
+		sourcePath, evalErr := filepath.EvalSymlinks(m.Source)
```

Comment:
This is what I had originally, but I changed it because the return variable names are exposed in the Godoc. See: https://github.com/moby/moby/pull/45687#discussion_r1245632003